### ISSUE TITLE
Lack of .shed.yml file for a tool or set of tools results in action deciding to do nothing but not giving an error

### ISSUE THREAD

**Issue by pcm32** [galaxyproject/planemo-ci-action#53]
Thanks again for this great action, we use it constantly.

I've noticed that if the .shed.yml file is not present in the directory of modified tools, the action doesn't fail but following steps don't take care of the tools. Maybe this is an issue on our CI setup, but I wonder if it would be better if part that determines commit ranges, repos and chunks fails instead alerting the user of the lack of a .shed.yml file? I took me a bit of time to realise it was missing.

**Comment by nsoranzo** [comment]
The `.shed.yml` file is used to identify directories that should contain tools/workflows from directories to ignore, so I don't see how we can support this unfortunately.

**Comment by pcm32** [comment]
No, of course, you cannot support it, I meant it should fail more explicitly rather than not doing anything in the following steps. From the outside, it looks like everything is successful + skipped = succes, which is misleading.

**Comment by bernt-matthias** [comment]
Actually we have such a check https://github.com/galaxyproject/planemo-ci-action/blob/fd4d12c417215f7cd9075227e784f4cd61c60044/planemo_ci_actions.sh#L126 

I would hope that this fixes it? https://github.com/galaxyproject/planemo-ci-action/pull/55
